pkg/webhook/v1alpha3/pod/const.go

```diff
@@ -20,10 +20,12 @@ const (
 	// JobNameLabel represents the label key for the job name, the value is job name
 	JobNameLabel = "job-name"
 	// JobRoleLabel represents the label key for the job role, e.g. the value is master
-	JobRoleLabel        = "job-role"
-	TFJobRoleLabel      = "tf-job-role"
-	PyTorchJobRoleLabel = "pytorch-job-role"
-	MasterRole          = "master"
+	JobRoleLabel                 = "job-role"
```

Comment:
Can we create an issue to track these labels so that these can be removed in the next release?

Reply:
sure, I can create it right now

Reply:
#746